### PR TITLE
bump hcl2 version

### PR DESCRIPTION
**Pull request by gruebel**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

- bumped the `hcl2` version to latest one and adjusted some tests, the new output looks better I would say 😄 
- I also limited the `networkx` version, so the newest release won't be picked up, because they officially don't support Python 3.7 anymore
- some other minor adjustments, which I stumbled upon

### CODE REVIEW

File: tests/terraform/graph/variable_rendering/expected_data.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
             "version": ["1.19"],
             "vpc_config": {
                 "security_group_ids": ["aws_security_group.master.id"],
-                "subnet_ids": "Tree('full_splat_expr_term', ['aws_subnet.eks', 'id'])"
+                "subnet_ids": "aws_subnet.eks[*].id"
```
Comment (nimrodkor) [inline tests/terraform/graph/variable_rendering/expected_data.py:34]: @gruebel Very cool that we now support it. How do we parse it as graph node attributes? As a string?

Comment (gruebel) [reply tests/terraform/graph/variable_rendering/expected_data.py:34]: our Transformer in the `hcl2` lib was missing a method, that's why it looked so weird. let me check, how it looks like in the graph node 🧐

Comment (gruebel) [reply tests/terraform/graph/variable_rendering/expected_data.py:34]: @nimrodkor currently it is just a string

**Comment by Saarett** [comment]
Nice!

**Review by Saarett — APPROVED** [review]

**Pull request merged**